formats/manual_section/publisher/details.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "body",
+    "manual",
+    "organisations"
+  ],
+  "properties": {
```

Comment:
Looks like the exporter is currently sending empty arrays for `breadcrumbs` and `child_section_groups` for the manual_sections format, which would be invalid according to this schema.